pkg/apis/openebs.io/v1alpha1/cstor_volume.go

```diff
@@ -82,7 +82,7 @@ type CVStatus struct {
 // ReplicaStatus stores the status of replicas
 type ReplicaStatus struct {
 	ID                string `json:"replicaId"`
-	Status            string `json:"status"`
+	Mode              string `json:"mode"`
```

Comment:
This seems to be a breaking change?
Can we have both the fields for backward compatibility?

Reply:
end users won't be using this. So, should be fine.
But, need to find a way of not to have frequent changes in this.

Reply:
yes we are not using replica-status any where , its just a part of the CVR object.